services/web/server/src/simcore_service_webserver/application_settings.py

```diff
@@ -250,57 +279,63 @@ class ApplicationSettings(BaseCustomSettings, MixinLoggingSettings):
         "Currently this is a system plugin and cannot be disabled",
     )
 
-    @root_validator()
+    model_config = SettingsConfigDict(
+        frozen=False
```

Comment:
why not frozen? Is there a reason to modify the settings once they are constructed?